platform/src/abstractions/EducationBenchmarking.Platform.Infrastructure/Cosmos/CosmosClientFactory.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.Azure.Cosmos;
+
+namespace EducationBenchmarking.Platform.Infrastructure.Cosmos;
+
+public static class CosmosClientFactory
+{
+    public static CosmosClient Create(string connectionString)
+    {
+        return new CosmosClient(connectionString, new CosmosClientOptions
+        {
+#if DEBUG
+            //Disabling SSL validation for local emulator
+            HttpClientFactory = () =>
+            {
+                HttpMessageHandler httpMessageHandler = new HttpClientHandler
+                {
+                    ServerCertificateCustomValidationCallback =
+                        HttpClientHandler.DangerousAcceptAnyServerCertificateValidator
```

Comment:
My understanding of `HttpClientHandler.DangerousAcceptAnyServerCertificateValidator` is that it allows you to use custom certificate verification, and while I can see the use for this for dev and test environments, it looks like it will introduce an area of risk in production. So, should we replace this with a different method to remove any risk from the beginning?

Reply:
`#if DEBUG ` is this a complier directive, such that when we release build (for dev, test and live) this code is not even present.

The CI/CD pipeline builds the code in release mode. The only time we'd you debug is locally.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives